tedium.js

```diff
@@ -181,12 +228,52 @@ function connectToGithub() {
   return github;
 }
 
+function analyzeRepos() {
+  const dirs = fs.readdirSync('repos/');
```

Comment:
Because ~~I'm lazy~~ these dirs were just created, so accessing them should be super fast, and we have no other work to do at this point in the program anyways.

This function returns a promise so we can later change this to async if we want without any pain.